README.md

```diff
@@ -1,106 +1,112 @@
--  Follow **all details** on [how to setup flawlessly](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) docker for windows and [WSL]
-
-
-## Releases
-
-- [Git release workflow](ops/README.md)
-- Public [releases](https://github.com/ITISFoundation/osparc-simcore/releases)
-- Production in https://osparc.io
-- [Staging instructions](docs/staging-instructions.md)
-
-
-
-
-<!-- ADD REFERENCES BELOW AND KEEP THEM IN ALPHABETICAL ORDER -->
-[chocolatey]:https://chocolatey.org/
-[vscode]:https://code.visualstudio.com/
-[WSL]:https://docs.microsoft.com/en-us/windows/wsl/faq
+# osparc-simcore platform
+
+**WARNING** This application is **still under development**.
+
+<!-- NOTE: when branched replace `master` in urls -->
+[`master`](https://github.com/itisfoundation/osparc-simcore/tree/master)
+[![Requires.io]](https://requires.io/github/ITISFoundation/osparc-simcore/requirements/?branch=master "State of third party python dependencies")
+[![travis-ci]](https://travis-ci.org/ITISFoundation/osparc-simcore "State of CI: build, test and pushing images")
+[![coverals.io]](https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master "Test coverage")
+[![github.io]](https://itisfoundation.github.io/)
+[![itis.dockerhub]](https://hub.docker.com/u/itisfoundation)
+
+
+<!-- THIS SERVICE MALFUNCTIONS
+[![webserver]](https://microbadger.com/images/itisfoundation/webserver "More on itisfoundation/webserver:staging-latest image")
+[![director]](https://microbadger.com/images/itisfoundation/director "More on itisfoundation/director:staging-latest image")
+[![sidecar]](https://microbadger.com/images/itisfoundation/sidecar "More on itisfoundation/sidecar:staging-latest image")
+[![storage]](https://microbadger.com/images/itisfoundation/storage "More on itisfoundation/storage:staging-latest image")
+-->
+
+
+<!-- ADD HERE ALL BADGE URLS -->
+[Requires.io]:https://img.shields.io/requires/github/ITISFoundation/osparc-simcore.svg
+[travis-ci]:https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master
+[coverals.io]:https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master
+[github.io]:https://img.shields.io/website-up-down-green-red/https/itisfoundation.github.io.svg?label=documentation
+[itis.dockerhub]:https://img.shields.io/website/https/hub.docker.com/u/itisfoundation.svg?down_color=red&label=dockerhub%20repos&up_color=green
+[webserver]:https://img.shields.io/microbadger/image-size/itisfoundation/webserver/staging-latest.svg?label=webserver&style=flat
+[director]:https://img.shields.io/microbadger/image-size/itisfoundation/director/staging-latest.svg?label=director&style=flat
+[sidecar]:https://img.shields.io/microbadger/image-size/itisfoundation/sidecar/staging-latest.svg?label=sidecar&style=flat
+[storage]:https://img.shields.io/microbadger/image-size/itisfoundation/storage/staging-latest.svg?label=storage&style=flat
```

Comment:
hmm the link looks like it's pointing to staging-latest... @pcrespov is there any reason there? it should be master-latest or there should be a table with the different ones.
we have the following images:
master-latest | staging-latest | latest
they respectively correspond to the latest built master-,staging- and released versions

Reply:
same, I took care of this in #1482